client/layout/guided-tours/config-elements/step.js

```diff
@@ -101,7 +104,8 @@ export default class Step extends Component {
 			this.quitIfInvalidRoute( nextProps, nextContext );
 			this.skipIfInvalidContext( nextProps, nextContext );
 			this.scrollContainer.removeEventListener( 'scroll', this.onScrollOrResize );
-			this.scrollContainer = query( nextProps.scrollContainer )[ 0 ] || global.window;
+			this.scrollContainer =
+				query( nextProps.scrollContainer )[ 0 ] || ( typeof window !== 'undefined' && window );
```

Comment:
Is the `typeof window` check necessary? If `window` is undefined, then the app will crash on the very next line anyway (`addEventListener`).

Reply:
it may not be necessary. I know sometimes it is to prevent tests from failing.
seems like the `typeof` check isn't covering enough in this case. 

Since tests are all passing, I can remove this check and someone can add it back in if the future if we ever need it